Sources/PackageLoading/PackageBuilder.swift

```diff
@@ -105,6 +125,30 @@ extension Module.Error: FixableError {
     }
 }
 
+extension Module.Error.ModuleNameProblem : FixableError {
+    var error: String {
+        switch self {
+          case .emptyName:
+            return "the module name is empty"
```

Comment:
It is only impossible today because the module name always comes from the file system name, but when validating the module name, it seems reasonable.  If any higher level imposes an override.  I can remove it until that happens, over course, so that we can get complete test coverage.

Did I mention this PR is very much a work in progress. :-)